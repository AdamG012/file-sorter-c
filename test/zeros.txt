0000001
0000002
00000000000000000000034
000000000000000000000000000000000534
1
12
0132312
38293892839208390
98920829108
382190328930218
