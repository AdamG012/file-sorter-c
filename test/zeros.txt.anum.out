0000001
1
0000002
12
00000000000000000000034
000000000000000000000000000000000534
0132312
98920829108
382190328930218
38293892839208390
